matt-rich-anderson/QuidProGo
# Ma create consultation

#### Changes Made
1. Added file `Create.cshtml` to `Views` directory.
​2. Added file `CategoryRepositories` to `Repositories` directory.

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout ma-createConsultation
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > As a logged in user, click "My Consultations"
    > Fill out the form, (you may select more than one category), hit submit
    > You show be returned the index where the new consultation should be present
